src/commands/git_secret_removeperson.sh

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
 
 
-function killperson {
+function removeperson {
```

Comment:
Let's add something like:

```bash
function killperson {
   echo 'WARNING: `killperson` is renamed to `removeperson`, it will be removed in the next version, please upgdare!'

   remove_person $@
}
```

Reply:
Sure, I have added that in b186d6c8deed4290b77b1117e61c80cbcbf44e84.

Reply:
I think we should leave` killperson` in indefinitely, and not deprecate it, so old scripts continue to work as expected.
I'm fine with updating all the docs to not mention `killperson` any more though.

Also while we're considering this, it's always bugged me a little that you `tell` a user and not `tellperson` or `addperson` (which would be more symmetric with `removeperson`). 

What do you all think about renaming the command  `addperson` (still supporting `tell` of course) and stop documenting `tell`?

Reply:
@joshrabinowitz I have changed the warning message. I think that we are free not to do this at the next version, but just sometime in the future.

Let's open a new issue for your `tellperson` proposal.